lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -227,17 +219,18 @@ def _import_genbank_mass(self, params):
         return details
 
     def _save_genomes(self, workspace_id, genome_objs):
-        results = [
-            self.gi.save_one_genome(
-                {
-                    'workspace': workspace_id,
-                    'name': genome_obj.genome_name,
-                    'data': genome_obj.genome_data,
-                    "meta": genome_obj.genome_meta,
-                }
-            ) for genome_obj in genome_objs
-        ]
-
+        results = self.gi.save_genome_mass(
```

Comment:
Are there any other opportunities for batching in this method or is it done at this point?

Reply:
Batching will be done in GenbankToGenome, so there is no need in save_genome_mass.

Reply:
:+1: